packages/forms/src/UIForm/fields/TimezoneList/TimezoneList.utils.js

```diff
@@ -0,0 +1,151 @@
+import get from 'lodash/get';
+
+import frTimezones from 'cldr-dates-full/main/fr/timeZoneNames.json';
+import enTimezones from 'cldr-dates-full/main/en/timeZoneNames.json';
+import jaTimezones from 'cldr-dates-full/main/ja/timeZoneNames.json';
+import deTimezones from 'cldr-dates-full/main/de/timeZoneNames.json';
```

Comment:
This might not be the best idea, to load all translations files by default (uncompressed, this is 4 * ~70 = 240kb)

Reply:
Do we need all the content of theses json ?
https://unpkg.com/cldr-dates-full@39.0.0/main/fr/timeZoneNames.json

Reply:
Exemplar city is the key we need actually, it's where the city name is translated